rosidl_generator_py/package.xml

```diff
@@ -20,12 +24,11 @@
   <buildtool_export_depend>rosidl_typesupport_c</buildtool_export_depend>
   <buildtool_export_depend>rosidl_typesupport_interface</buildtool_export_depend>
 
-
   <exec_depend>python3-numpy</exec_depend>
   <exec_depend>rmw_implementation</exec_depend>
   <exec_depend>rmw_implementation_cmake</exec_depend>
-  <exec_depend>rosidl_generator_c</exec_depend>
```

Comment:
I doubt this can be removed since it is being used here: https://github.com/ros2/rosidl_python/blob/a98f9378ce345dbb4fe628723af8273210ee004a/rosidl_generator_py/cmake/rosidl_generator_py_generate_interfaces.cmake#L17

Reply:
This was replace by `rosidl_runtime_c` https://github.com/ros2/rosidl_python/blob/dea8232a1e1a9b9c92d7c8b15dfcd2a215ae0a6f/rosidl_generator_py/cmake/rosidl_generator_py_generate_interfaces.cmake#L17

https://github.com/ros2/rosidl_python/blob/dea8232a1e1a9b9c92d7c8b15dfcd2a215ae0a6f/rosidl_generator_py/cmake/rosidl_generator_py_generate_interfaces.cmake#L266-L270

https://github.com/ros2/rosidl_python/blob/dea8232a1e1a9b9c92d7c8b15dfcd2a215ae0a6f/rosidl_generator_py/cmake/rosidl_generator_py_generate_interfaces.cmake#L280-L283